src/java.base/share/classes/java/nio/file/Files.java

```diff
@@ -1408,8 +1408,9 @@ public static Path copy(Path source, Path target, CopyOption... options)
      * @throws  DirectoryNotEmptyException
      *          the {@code REPLACE_EXISTING} option is specified but the file
      *          cannot be replaced because it is a non-empty directory, or the
-     *          source is a non-empty directory containing entries that would
-     *          be required to be moved <i>(optional specific exceptions)</i>
+     *          source is a non-empty directory containing entries that would be
+     *          required to be moved and the target location is on a different
+     *          {@code FileStore} <i>(optional specific exceptions)</i>
      * @throws  AtomicMoveNotSupportedException
```

Comment:
I'm not sure about this change to the spec because there may be other reasons why entries might have to be moved, esp. when you get into exotic/custom file systems.

Reply:
So do you think try to refine this or withdraw the PR and manually resolve the issue?

Reply:
Yes, I think we should just close the issue, assuming the exception is clear.

Reply:
Thanks. I'll double check it on Windows 11 and then close these.